e2e-tests/tfm/cypress/e2e/journeys/deal-cancellation/reason-for-cancelling.spec.js

```diff
@@ -75,6 +74,16 @@ context('Deal cancellation - reason for cancelling', () => {
 
       cy.url().should('eq', relative(`/case/${dealId}/cancellation/cancel`));
     });
+
+    it('returning to the page should display saved data', () => {
+      const reason = 'A Reason';
+      cy.keyboardInput(reasonForCancellingPage.reasonForCancellingTextBox().clear(), reason);
```

Comment:
Need to remove `.clear()` here, test is probably failing as is @BethThomas141 🙏

Reply:
Lets get this merged, I will pick this up in [my PR](https://github.com/UK-Export-Finance/dtfs2/pull/3541) 

I don't think the deal cancellation tests are running in GHA - otherwise this test should be failing because of the `.clear()`

```
'./tfm/cypress/e2e/journeys/deal-cancellation/**/*.spec.js'

to 

'./tfm/cypress/e2e/journeys/deal-cancellation/*.spec.js'
```

Or we could do this in the next PR?

Reply:
Ooh interesting, so I've removed it, but I don't think the `.clear()` does impact whether or not the test passes/fails. 

It has the right structure in the `test.yml` file, it's consistent with the other similar subfolders, and the tests are indeed running:

https://github.com/UK-Export-Finance/dtfs2/actions/runs/11109438117/job/30864938680?pr=3571

![image](https://github.com/user-attachments/assets/aa46a53a-f689-48c4-8e72-d4116e1b88de)

So I think it's just a case that `.clear()` isn't needed but won't stop a test passing. Either way, have removed it here 👍

Reply:
Thank you @BethThomas141 , this seems very strange as in another PR, `.clear()` stopped the test from passing, 

`cy.keyboardInput` (correctly) fails if a selector is not passed, and `.clear()` is not a selector, rather it's a method on a selector

I would like to run the test locally with `.clear()` to see if it passes but i'm currently unable to do so.

Anyway, i've removed this in another PR